test/unit/util/project-loader-hoc.test.jsx

```diff
@@ -1,51 +1,57 @@
 import React from 'react';
-import ProjectLoaderHOC, {ProjectLoader} from '../../../src/lib/project-loader-hoc.jsx';
+import ProjectLoaderHOC from '../../../src/lib/project-loader-hoc.jsx';
+import storage from '../../../src/lib/storage';
 import {mount} from 'enzyme';
 
 describe('ProjectLoaderHOC', () => {
     test('when there is no project data, it renders null', () => {
         const Component = ({projectData}) => <div>{projectData}</div>;
         const WrappedComponent = ProjectLoaderHOC(Component);
         window.location.hash = '#winning';
-        ProjectLoader.load = jest.fn((id, cb) => cb(null, null));
+        storage.load = jest.fn(() => Promise.resolve(null));
         const mounted = mount(<WrappedComponent />);
-        ProjectLoader.load.mockRestore();
+        storage.load.mockRestore();
```

Comment:
I'm not sure if `mockRestore` works this way. The documentation says:
> Beware that `mockFn.mockRestore` only works when mock was created with `jest.spyOn`. Thus you have to take care of restoration yourself when manually assigning `jest.fn()`.
https://facebook.github.io/jest/docs/en/mock-function-api.html#mockfnmockrestore

Reply:
Thanks!